src/proxy/mod.rs

```diff
@@ -208,3 +208,5 @@ pub fn to_canonical_ip(ip: SocketAddr) -> IpAddr {
         },
     }
 }
+
+const ERR_TOKIO_RUNTIME_SHUTDOWN: &str = "A Tokio 1.x context was found, but it is being shutdown.";
```

Comment:
nit: would expose this as `fn is_runtime_shutdown(e)`

Reply:
SG